packages/zosfiles/src/doc/IOptions.ts

```diff
@@ -57,4 +58,8 @@ export interface IOptions extends IZosFilesOptions {
      * @type {ITaskWithStatus}
      */
     task?: ITaskWithStatus;
+    /**
+     * The path to a .zosattributes file used to control file conversion and tagging.
+     */
+    attributes?: ZosFilesAttributes;
```

Comment:
Is the `attributes` property needed on the `IOptions` interface? I believe that adding it to `IDownloadOptions` should be adequate for the changes in this PR.